README.md

```diff
@@ -49,7 +54,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:castle][:guests] += [town[:residents].slice!(1)]
```

Comment:
:+1:  Looks good to me; or the very similar `town[:castle][:guests] << town[:residents].slice!(1)`